db/log_test.cc

```diff
@@ -286,6 +298,12 @@ TEST_P(LogTest, Fragmentation) {
 }
 
 TEST_P(LogTest, MarginalTrailer) {
+  const bool compression_enabled =
+      std::get<2>(GetParam()) == kNoCompression ? false : true;
+  // Only run if WAL compression is not enabled.
```

Comment:
You're invoking this exclusion pattern too many times. Please create separate parameterized test class that never includes compression.

Reply:
I created a new class that supports compression tests that inherits from the base class. Not sure if this is the best way to implement it.